<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>100</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Tactile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../tactile2.qrc">
    <normaloff>:/res/icons/tactile2_icon2.png</normaloff>:/res/icons/tactile2_icon2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_map"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_map"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_row"/>
    <addaction name="actionAdd_column"/>
    <addaction name="actionRemove_row"/>
    <addaction name="actionRemove_column"/>
    <addaction name="separator"/>
    <addaction name="actionStamp_tool"/>
    <addaction name="actionBucket_tool"/>
    <addaction name="actionEraser_tool"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCenter_camera"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_grid"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="separator"/>
    <addaction name="actionPan_up"/>
    <addaction name="actionPan_down"/>
    <addaction name="actionPan_right_3"/>
    <addaction name="actionPan_left"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Tactile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_map"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_map"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew_map">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/file.png</normaloff>:/res/icons/icons8/color/64/file.png</iconset>
   </property>
   <property name="text">
    <string>New map</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_map">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/opened-folder.png</normaloff>:/res/icons/icons8/color/64/opened-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open map...</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/save.png</normaloff>:/res/icons/icons8/color/64/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/save-as.png</normaloff>:/res/icons/icons8/color/64/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/rename.png</normaloff>:/res/icons/icons8/color/64/rename.png</iconset>
   </property>
   <property name="text">
    <string>Rename...</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/settings.png</normaloff>:/res/icons/icons8/color/64/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/exit.png</normaloff>:/res/icons/icons8/color/64/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/undo.png</normaloff>:/res/icons/icons8/color/64/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/redo.png</normaloff>:/res/icons/icons8/color/64/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionAdd_row">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/add-row.png</normaloff>:/res/icons/icons8/color/64/add-row.png</iconset>
   </property>
   <property name="text">
    <string>Add row</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAdd_column">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/add-column.png</normaloff>:/res/icons/icons8/color/64/add-column.png</iconset>
   </property>
   <property name="text">
    <string>Add column</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionRemove_row">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/remove-row.png</normaloff>:/res/icons/icons8/color/64/remove-row.png</iconset>
   </property>
   <property name="text">
    <string>Remove row</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionRemove_column">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/remove-column.png</normaloff>:/res/icons/icons8/color/64/remove-column.png</iconset>
   </property>
   <property name="text">
    <string>Remove column</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCenter_camera">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/center-camera.png</normaloff>:/res/icons/icons8/color/64/center-camera.png</iconset>
   </property>
   <property name="text">
    <string>Center camera</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionToggle_grid">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/grid.png</normaloff>:/res/icons/icons8/color/64/grid.png</iconset>
   </property>
   <property name="text">
    <string>Toggle grid</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/zoom-in.png</normaloff>:/res/icons/icons8/color/64/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/zoom-out.png</normaloff>:/res/icons/icons8/color/64/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionPan_right">
   <property name="text">
    <string>Pan up</string>
   </property>
  </action>
  <action name="actionPan_right_2">
   <property name="text">
    <string>l</string>
   </property>
  </action>
  <action name="actionPan_up">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/arrow-up.png</normaloff>:/res/icons/icons8/color/64/arrow-up.png</iconset>
   </property>
   <property name="text">
    <string>Pan up</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionPan_right_3">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/arrow-right.png</normaloff>:/res/icons/icons8/color/64/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Pan right</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionPan_left">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/arrow-left.png</normaloff>:/res/icons/icons8/color/64/arrow-left.png</iconset>
   </property>
   <property name="text">
    <string>Pan left</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionPan_down">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/arrow-down.png</normaloff>:/res/icons/icons8/color/64/arrow-down.png</iconset>
   </property>
   <property name="text">
    <string>Pan down</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionAbout_Tactile">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/about.png</normaloff>:/res/icons/icons8/color/64/about.png</iconset>
   </property>
   <property name="text">
    <string>About Tactile...</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionStamp_tool">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/brush.png</normaloff>:/res/icons/icons8/color/64/brush.png</iconset>
   </property>
   <property name="text">
    <string>Stamp tool</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionBucket_tool">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/bucket.png</normaloff>:/res/icons/icons8/color/64/bucket.png</iconset>
   </property>
   <property name="text">
    <string>Bucket tool</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionEraser_tool">
   <property name="icon">
    <iconset resource="../tactile2.qrc">
     <normaloff>:/res/icons/icons8/color/64/eraser.png</normaloff>:/res/icons/icons8/color/64/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Eraser tool</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../tactile2.qrc"/>
 </resources>
 <connections/>
</ui>
